CardEngine/src/TurnSystem/Events/endTurnEventData.h

```diff
@@ -0,0 +1,12 @@
+﻿#pragma once
+
+namespace TurnSystem
```

Comment:
There's no consistency in naming convention. I highly suggest in having one

Reply:
yeah, I noticed a lot of inconsistencies this morning, most on namespaces, already fixing it in the new branch